css/jquery-confirm.css

```diff
@@ -517,3 +517,73 @@ body.jconfirm-noscroll {
   margin: 0;
   border-left: solid 1px #ddd;
 }
+
+
+.jcbutton {
```

Comment:
Yes made bootstrap optional now, now I'm adding buttons now, with help of your CSS. Will push the changes in a few hours.